### DISCUSSION TITLE
Welcome to epv-api-scripts Discussions!

### DISCUSSION THREAD

**Discussion by bab29** [cyberark/epv-api-scripts#229] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lnhquang1993** [comment]
Hi, I am using CyberArk version 13.2. Does the script Get-Account still valid for this version? Because i try to run the script but not working.

**Reply by bab29** [reply]
Please open an issue including the logs when you run the command with -verbose
